server/router.js

```diff
@@ -125,7 +130,9 @@ function sendSPA (route, req, res, params, state) {
       return res.end(require('./page-debug')(contents, serializeJS(frozenState)))
     })
   }
-  return res.end(page(contents, serializeJS(frozenState)))
+  if (!metadata) metadata = {}
+  metadata.route = route
+  return res.end(page(contents, metadata, serializeJS(frozenState)))
```

Comment:
and by state, i mean in the `contents` argument. it is an archive model property.